content/the-ember-times-issue-182.md

```diff
@@ -2,27 +2,32 @@
 title: The Ember Times - Issue No. 182
 authors:
   - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+  - chris-ng
 date: 2021-06-18T00:00:00.000Z
 tags:
   - newsletter
   - '2021'
 ---
 
 👋 Emberistas! 🐹
-
+Read the blog post on using the in-element helper ⛳,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [1. Section title in sentence case 🐹](section-url)
+## [Blog Post: Using the in-element helper ⛳](https://www.linkedin.com/pulse/emberjs-using-in-element-helper-faith-or/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Faith Or (@faith-or)](https://github.com/faith-or) wrote a blog post on how they migrated from `ember-wormhole` to the built-in `in-element` helper introduced in Ember 3.20.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+[Ember Wormhole](https://github.com/yapplabs/ember-wormhole) is an addon that renders a child view somewhere else in the DOM.
+
+The [in-element helper](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) renders its block content outside of the regular flow, into a DOM element given by its `destinationElement` positional argument.
+
+Common use cases for both include when a piece of UI is a logical child of a component but needs to render somewhere else such as a loading screen, a modal, tooltip or even a dropdown. The differences between the two are slight. Faith goes into more detail in the blog post.
+
+The main takeaways is that  `in-element` needs the destination to exist before we use it while `ember-wormhole` does not, when the destination element changes the content is re-rendered completely for `in-element`, and by default the `in-element` helper will replace all the contents of the destination element (you will need to configure it not to using `insertBefore=null`).
```

Comment:
I think the paragraph will be easier to read if we make a list and specify the number of takeaways in the beginning.

```suggestion
There are 3 main takeaways.

1. `in-element` needs the destination to exist before we use it, while `ember-wormhole` does not.
2. With `in-element`, when the destination element changes, the content is re-rendered completely.
3. By default, the `in-element` helper will replace all the contents of the destination element. (You will need to configure it not to using `insertBefore=null`.)
```